types/trigger.d.ts

```diff
@@ -173,6 +173,8 @@ type RequiredFieldsAsUnion<Type> = {
 export type BaseTriggerSet<Data extends RaidbossData> = {
   // ZoneId.MatchAll (aka null) is not supported in array form.
   zoneId: ZoneIdType | number[];
+  // useful if the zoneId is an array or zone name is otherwise non-descriptive
+  zoneLabel?: LocaleText;
```

Comment:
Can you add a note about this to https://github.com/quisquous/cactbot/blob/main/docs/RaidbossGuide.md#file-structure?

Reply:
done.